controller/work_item_children_blackbox_test.go

```diff
@@ -155,7 +155,7 @@ func (s *workItemChildSuite) SetupTest() {
 	require.Nil(s.T(), err)
 	s.T().Logf("Created bug2 with ID: %s\n", *bug2.Data.ID)
 
-	bug3Payload := CreateWorkItem(s.userSpaceID, workitem.SystemBug, "bug3")
+	bug3Payload := getCreateWorkItemPayload(s.userSpaceID, workitem.SystemBug, "bug3")
```

Comment:
a `getCreate` function name is kind of a weird name. What about `createWorkItemPayload`. Also that is a golang recommendation to avoid the prefix setter/getter but we have this all over the code :D.